examples/shap/explainer_logging.py

```diff
@@ -0,0 +1,31 @@
+import mlflow
+import shap
```

Comment:
You will need to move the shap and sklearn imports into the associated functions to avoid early imports of optional packages